R/find.tol.time.R

```diff
@@ -207,7 +207,7 @@ find.tol.time <- function(features,
     }
 
     list(
-        features = features,
+        features = features |> dplyr::select(-mz_group),
```

Comment:
Why has this changed?

Reply:
Since basically a single tibble is passed through, this column doesn't get removed before further steps anymore.